packages/beacon-node/test/unit/api/impl/node/node.test.ts

```diff
@@ -43,36 +41,6 @@ describe("node api implementation", function () {
     sinon.stub(networkStub, "localMultiaddrs").get(() => [multiaddr("/ip4/127.0.0.1/tcp/36000")]);
   });
 
-  describe("getNetworkIdentity", function () {
```

Comment:
Does this unit test need to be reimplemented?

Reply:
The API impl now just calls a function, a unit test for it would be pretty stupid, testing that function a calls function b